### PR TITLE
Api play

### PR DESCRIPTION
**Pull request by Stearnzy**
Faraday added
Figaro added and used to hide API key
Movie controller made with index for the Search for Movies capability
Flash messages for login changed
Tests for login spec recategorized
Tests written for search spec
STILL NEED TESTS FOR API

### CODE REVIEW

File: app/views/movies/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h2><%= button_to 'Find Top Rated Movies', '/movies' %></h2>
+
+<%= form_with url: '/movies', method: 'get', local: true do |f| %>
+  <%= f.text_field :movie_title , placeholder: "Search by movie title"%>
+  <%= f.submit "Find Movies" %>
+<% end %>
+
+<% @titles.each do |title| %>
+  <div class="movie-column">
+    <ul class="movie-results"><%= title[:original_title] %></ul>
```
Comment (stillsheryl) [inline app/views/movies/index.html.erb:10]: Maybe we can turn movies into objects during a refactor so we don't have to use a hash here.

File: spec/features/movies/search_spec.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      within all('.movie-results')[22] do
+        expect(page).to have_content(/star/i)
+      end
+
+      within all('.movie-results')[39] do
+        expect(page).to have_content(/star/i)
+      end
+    end
+
+    it "I see no more than 40 results" do
```
Comment (stillsheryl) [inline spec/features/movies/search_spec.rb:35]: Nice edge case test!

**Review by stillsheryl — APPROVED** [review]
Looks good!

**Review by Eternal-Flame085 — APPROVED** [review]
The second Travis gem seems redundant
